src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.scss

```diff
@@ -1,82 +1,89 @@
+      margin-left: 25px;
+      width: calc(100% - 50px);
+    }
+
+    .sidebar-slider {
+      position: absolute;
+      background-color: var(--ubs-primary-grey);
+      margin-top: 80px;
+    }
   }
-}
 
-.main-content {
-  margin-top: 120px;
-  overflow-x: hidden;
-}
+  .main-content {
+    margin-top: 120px;
+    overflow-x: hidden;
+    min-height: 345px;
+  }
 
-.sidebar-list {
-  margin: 0;
-  padding: 0;
-  list-style-type: none;
-  text-align: center;
+  .sidebar-list {
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+    text-align: center;
 
-  .sidebar-list-item {
-    display: flex;
-    align-items: center;
-    column-gap: 20px;
-    cursor: pointer;
-    padding: 15px 100px 15px 20px;
+    .sidebar-list-item {
+      display: flex;
+      align-items: center;
+      column-gap: 20px;
+      cursor: pointer;
+      padding: 15px 39px 15px 20px;
 
-    &:hover,
-    &:focus {
-      background-color: var(--ubs-secondary-grey);
+      &:hover,
+      &:focus {
+        background-color: var(--ubs-secondary-grey);
+      }
     }
   }
-}
 
-.sidebar-list-item-link {
-  color: var(--ubs-primary-white);
-  font-family: var(--primary-font);
-  font-size: 16px;
-  line-height: 1.5em;
-  text-decoration: none;
-}
+  .sidebar-list-item-link {
+    color: var(--ubs-primary-white);
+    font-family: var(--primary-font);
+    font-size: 16px;
+    line-height: 1.5em;
+    text-decoration: none;
+  }
 
-.sidebar-icons {
-  position: fixed;
-  width: 3.7em;
-  height: 100vh;
-  margin-top: 80px;
-  background-color: var(--ubs-primary-grey);
-  -webkit-user-select: none;
-  -moz-user-select: none;
-  -ms-user-select: none;
-  user-select: none;
-}
+  .sidebar-icons {
+    position: absolute;
+    width: 3.7em;
+    height: 100%;
+    padding-top: 80px;
+    background-color: var(--ubs-primary-grey);
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    z-index: 1;
+  }
 
-.sidebar-toggler {
-  width: 25px;
-  height: 25px;
-  border-radius: 50%;
-  border: 1px solid var(--ubs-primary-green);
-  float: right;
-  background-color: var(--ubs-primary-grey);
-  -webkit-user-select: none;
-  -moz-user-select: none;
-  -ms-user-select: none;
-  user-select: none;
-  display: flex;
-  align-items: center;
-  margin: 7px;
-}
+  .sidebar-toggler {
+    width: 25px;
+    height: 25px;
+    border-radius: 50%;
+    border: 1px solid var(--ubs-primary-green);
+    float: right;
+    background-color: var(--ubs-primary-grey);
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    display: flex;
+    align-items: center;
+    margin: 7px;
+  }
 
-.arrow {
-  width: 15px;
-}
+  .arrow {
+    width: 15px;
+  }
 
-.count-unread-messages {
-  color: var(--ubs-primary-white);
-  margin: 0 -25px 0 -10px;
+  .count-unread-messages {
+    color: var(--ubs-primary-white);
+    margin: 0 -25px 0 -10px;
+  }
```

Comment:
You shouldn't change the indentations
Please leave the only changes related to your task

Reply:
Done